spec/factories/users.rb

```diff
@@ -1,10 +1,10 @@
 FactoryBot.define do
   factory :user do
-    email {'user@mail.com'}
+    email { Faker::Internet.email }
```

Comment:
Why do we use this here? I have no problem with it, I am just interested :)

Reply:
If you dont want to use or dont have faker gem, you can write the email line like this
```
    sequence(:email) { |n| "email#{n}@mail.com" }
```
this will output
```
email1@mail.com
email2@mail.com
email3@mail.com
.....
```

Reply:
thanks for the valuable information, i will continue to use faker here but will try the other method next time ;)